tests/console/zziplib.pm

```diff
@@ -17,37 +17,45 @@ use strict;
 use warnings;
 use testapi;
 use utils;
+use version_utils 'is_sle';
 
 sub run {
     my $filezip = "files.zip";
     select_console "root-console";
-    # create a tmp dir/files to work
-    assert_script_run "mkdir /tmp/zip; cp /usr/share/doc/* /tmp/zip -R";
-    assert_script_run "cd /tmp";
-
-    # install requirements
-    zypper_call "in libzzip-0-13 zziplib-devel zip";
-
-    # create a zip file
-    assert_script_run "zip -9 $filezip -xi zip/*";
-    # Use unzip-mem on zip file to list archived files (-l)
-    assert_script_run "unzip-mem -l $filezip";
-
-    # Use unzip-mem on zip file to get a verbose list archived files (-v)
-    # Option -v  creates a core dump(bsc#1129403), create a soft-failure
-    my $RETURN_CODE = script_run("unzip-mem -v $filezip");
-    if (($RETURN_CODE) eq "0") {
-        assert_script_run("unzip-mem -v $filezip");
-    } else {
-        record_soft_failure "bsc#1129403 - Option -v creates a core dump";
-        save_screenshot;
+
+    # SLES 12 SP1 and Sp2 need add repo SDK manually, if case, skip test.
+    if (is_sle('>12-SP2')) {
```

Comment:
Please do it in main_common.pm in similar way  https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/41c61212eb62f9d09aa2284fc8ebf9c0fedabaca/lib/main_common.pm#L1611

Reply:
Dammit!!! ok, changed. Thank you.